src/bin/stress-test/types.ts

```diff
@@ -5,19 +5,34 @@ import {
   asNumber,
   asObject,
   asString,
-  asUnknown,
   asValue
 } from 'cleaners'
 
+export interface ErrorObj extends Error {
+  [key: string]: any
+}
+export const asErrorObj = (raw: any): ErrorObj => {
+  const clean = asObject({
```

Comment:
Once you've validated that this is an Error object, why not return it as-is? This copying seems unnecessary.

Reply:
The error needs to be serializable through IPC. Node's IPC API returns `{}` for the error object IIRC.

Reply:
Ok, that makes perfect sense then.